.pkg/DEBIAN/control

```diff
@@ -1,4 +1,4 @@
-Package: psst
+Package: psst-gui
```

Comment:
Is there a reason to rename this package?

Reply:
Yea #391

Reply:
https://github.com/jpochyla/psst/issues/391